core/src/main/java/org/axonframework/commandhandling/SimpleCommandBus.java

```diff
@@ -184,18 +211,99 @@ public Registration registerHandlerInterceptor(MessageHandlerInterceptor<? super
      * @return handle to unregister the interceptor
      */
     @Override
-    public Registration registerDispatchInterceptor(MessageDispatchInterceptor<? super CommandMessage<?>> dispatchInterceptor) {
+    public Registration registerDispatchInterceptor(
+            MessageDispatchInterceptor<? super CommandMessage<?>> dispatchInterceptor) {
         dispatchInterceptors.add(dispatchInterceptor);
         return () -> dispatchInterceptors.remove(dispatchInterceptor);
     }
 
     /**
-     * Sets the RollbackConfiguration that allows you to change when the UnitOfWork is committed. If not set the
-     * RollbackOnUncheckedExceptionConfiguration will be used, which triggers a rollback on all unchecked exceptions.
+     * Sets the {@link RollbackConfiguration} that allows you to change when the {@link UnitOfWork} is rolled back. If
+     * not set the, {@link RollbackConfigurationType#UNCHECKED_EXCEPTIONS} will be used, which triggers a rollback on
+     * all unchecked exceptions.
      *
-     * @param rollbackConfiguration The RollbackConfiguration.
+     * @param rollbackConfiguration a {@link RollbackConfiguration} specifying when a {@link UnitOfWork} should be
+     *                              rolled back
      */
     public void setRollbackConfiguration(RollbackConfiguration rollbackConfiguration) {
         this.rollbackConfiguration = rollbackConfiguration;
     }
+
+    /**
+     * Builder class to instantiate a {@link SimpleCommandBus}.
+     * <p>
+     * The {@link TransactionManager} is defaulted to a {@link NoTransactionManager}, the {@link MessageMonitor} is
+     * defaulted to a {@link NoOpMessageMonitor} and the {@link RollbackConfiguration} defaults to a
+     * {@link RollbackConfigurationType#UNCHECKED_EXCEPTIONS}.
+     * The {@link TransactionManager}, {@link MessageMonitor} and {@link RollbackConfiguration} are <b>hard
+     * requirements</b>. Thus setting them to {@code null} will result in an {@link AxonConfigurationException}.
+     */
+    public static class Builder {
+
+        private TransactionManager transactionManager = NoTransactionManager.INSTANCE;
+        private MessageMonitor<? super CommandMessage<?>> messageMonitor = NoOpMessageMonitor.INSTANCE;
+        private RollbackConfiguration rollbackConfiguration = RollbackConfigurationType.UNCHECKED_EXCEPTIONS;
+
+        /**
+         * Sets the {@link TransactionManager} used to manage transactions. Defaults to a {@link NoTransactionManager}.
+         *
+         * @param transactionManager a {@link TransactionManager} used to manage transactions
+         * @return the current Builder instance, for a fluent interfacing
```

Comment:
nit: not sure, so just asking..
Should it be ` for a fluent interfacing` or ` for fluent interfacing`

Reply:
....definitely a typo, kinda shitty this is like everywhere now.
Thanks anyhow, and thank yourself for increasing the PR size ;p